hudi-spark/src/main/java/org/apache/hudi/internal/HoodieDataSourceInternalWriter.java

```diff
@@ -102,7 +102,7 @@ public void commit(WriterCommitMessage[] messages) {
             .flatMap(m -> m.getWriteStatuses().stream().map(m2 -> m2.getStat())).collect(Collectors.toList());
 
     try {
-      writeClient.commitStats(instantTime, writeStatList, Option.empty());
+      writeClient.commitStats(instantTime, writeStatList, Option.empty(), HoodieTimeline.COMMIT_ACTION); //TODO get action type from HoodieWriterCommitMessage
```

Comment:
Are you planning to address this TODO as part of this PR ?

Reply:
Fixed